doc/security/vulnerabilities.rst

```diff
@@ -11,15 +11,380 @@ sufficient time to fix them.  Because the release notes are locked to
+code execution.
+
+This has been fixed in release v1.14.0.
+
+- `CVE-2017-14201 <https://docs.zephyrproject.org/latest/security/vulnerabilities.html#CVE-2017-14201>`_
+
+- `Zephyr project bug tracker ZEPSEC-17
+  <https://zephyrprojectsec.atlassian.net/browse/ZEPSEC-17>`_
+
+- `PR13260 fix for v1.14.0
+  <https://github.com/zephyrproject-rtos/zephyr/pull/13260>`_
+
+CVE-2017-14202
+--------------
+
+The shell implementation does not protect against buffer overruns
+resulting in unpredicable behavior.
+
+Improper Restriction of Operations within the Bounds of a Memory
+Buffer vulnerability in the shell component of Zephyr allows a serial
+or telnet connected user to cause a crash, possibly with arbitrary
+code execution.
+
+This has been fixed in release v1.14.0.
+
+- `CVE-2017-14202 <https://docs.zephyrproject.org/latest/security/vulnerabilities.html#CVE-2017-14202>`_
+
+- `Zephyr project bug tracker ZEPSEC-18
+  <https://zephyrprojectsec.atlassian.net/browse/ZEPSEC-18>`_
+
+- `PR13048 fix for v1.14.0
+  <https://github.com/zephyrproject-rtos/zephyr/pull/13048>`_
+
+CVE-2019
+========
+
+CVE-2019-9506
+-------------
+
+The Bluetooth BR/EDR specification up to and including version 5.1
+permits sufficiently low encryption key length and does not prevent an
+attacker from influencing the key length negotiation. This allows
+practical brute-force attacks (aka "KNOB") that can decrypt traffic
+and inject arbitrary ciphertext without the victim noticing.
+
+- `CVE-2019-9506 <http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-9506>`_
+
+- `Zephyr project bug tracker ZEPSEC-20
+  <https://zephyrprojectsec.atlassian.net/browse/ZEPSEC-20>`_
+
+- `PR18702 fix for v1.14.0
+  <https://github.com/zephyrproject-rtos/zephyr/pull/18702>`_
+
+- `PR18659 fix for v2.0.0
+  <https://github.com/zephyrproject-rtos/zephyr/pull/18659>`_
+
+CVE-2020
+========
+
+CVE-2020-10019
+--------------
+
+Buffer Overflow vulnerability in USB DFU of zephyr allows a USB
+connected host to cause possible remote code execution.
+
+This has been fixed in releases v1.14.2, v2.2.0, and v2.1.1.
+
+- `CVE-2020-10019 <http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-10019>`_
+
+- `Zephyr project bug tracker ZEPSEC-25
+  <https://zephyrprojectsec.atlassian.net/browse/ZEPSEC-25>`_
+
+- `PR23460 fix for 1.14.x
+  <https://github.com/zephyrproject-rtos/zephyr/pull/23460>`_
+
+- `PR23457 fix for 2.1.x
+  <https://github.com/zephyrproject-rtos/zephyr/pull/23457>`_
+
+- `PR23190 fix in 2.2.0
+  <https://github.com/zephyrproject-rtos/zephyr/pull/23190>`_
+
+CVE-2020-10021
+--------------
+
+Out-of-bounds write in USB Mass Storage with unaligned sizes
+
+Out-of-bounds Write in the USB Mass Storage memoryWrite handler with
+unaligned Sizes.
+
+See NCC-ZEP-024, NCC-ZEP-025, NCC-ZEP-026
+
+This has been fixed in releases v1.14.2, and v2.2.0.
+
+- `CVE-2020-10021 <https://docs.zephyrproject.org/latest/security/vulnerabilities.html#CVE-2020-10021>`_
```

Comment:
shouldn't it be `CVE-2020-10019 <http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-10021>`